### DISCUSSION TITLE
PLUS server windows 11 - missing dll

### DISCUSSION THREAD

**Discussion by aagrawal-x9** [PlusToolkit/PlusLib#1175] [Q&A]
I just installed the PLUS server for connecting with an Interson USB probe, but I am seeing an error message related to the missing WDAPI1010.dll. Has anyone else seen this error before and have a suggestion on possible resolution?

**Comment by Sunderlandkyl** [comment]
Information can be found here: 
http://perk-software.cs.queensu.ca/plus/doc/nightly/user/DeviceInterson.html

> Requires SDK provided by Interson for compilation and USB drivers provided by Interson for running (otherwise Plus application will not start due to missing WDAPI1010.dll).

**Reply by aagrawal-x9** [reply]
Thank you for the quick response. I am using the new SDK (received a couple months ago) and it is a C# .NET based project, and the drivers were installed to connect with the probe, which I am able to using the SDK application. Is there a different PLUS server application for old SDK that I could potentially try?

**Reply by aagrawal-x9** [reply]
@Sunderlandkyl - I looked into this more and found that the SDK version Interson PLUS version uses is now outdated and was compatible with the probes that are couple generations (~5 years) older.

Is this something the team might be able to add to backlog for an update? If not, do you have any suggestions for alternative options? I am just looking to relay b-mode stream from an interson probe to 3D Slicer via OpenIGT

**Reply by lassoan** [reply]
There are two interson SDK options in Interson, but probably both are quite old. You can rebuild PLUS with the new SDK. If everything works well then you can send a pull request to update PLUS (in case some changes are needed) and share the SDK files with us and then we may be able to update the nightly builds to use this more recent Interson SDK in the prebuilt binaries.
